docs/cloud-native-security/cloud-workload-protection.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[cloud-workload-protection]]
+= Cloud workload protection
+
+Cloud workload protection helps you monitor and understand what's happening on your Linux hosts. It uses the Elastic Defend integration to capture Linux process data, including commands executed and output generated during shell sessions.
+
+[discrete]
+== Capabilities
```

Comment:
Session-view and the dashboard are two views within CWP. I would not classify them as capabilities. 

We should make sure that tell the user that the CWP capabilities are really integrated with the entire Endpoint Defend product.  

- file, network, process telemetry 
- on-host detections and protections such as memory, malicious behaviour, malware. 
- Kibana based rules and ML 
- alert triage
- threat investigation
- Response actions and remediation 

Perhaps we can link to other places in the docs for these topics?